src/wallet/rpcwallet.cpp

```diff
@@ -379,7 +379,7 @@ UniValue getaddressesbyaccount(const JSONRPCRequest& request)
     return ret;
 }
 
-static void SendMoney(CWallet * const pwallet, const CTxDestination &address, CAmount nValue, bool fSubtractFeeFromAmount, CWalletTx& wtxNew, const CCoinControl& coin_control)
+static void SendMoney(CWallet * const pwallet, const CTxDestination &address, CAmount nValue, bool fSubtractFeeFromAmount, const CCoinControl& coin_control, mapValue_t mapValue, std::string fromAccount, CTransactionRef& txNew)
```

Comment:
Given that the argument `txNew` seems to be only used to store the resulting output in, wouldn't it be more convenient to make it the return type rather than an argument?

Reply:
> Given that the argument txNew seems to be only used to store the resulting output in, wouldn't it be more convenient to make it the return type rather than an argument?

Forgot to respond to this commit, but this was a good idea, and I implemented it in a previous update to the PR: a0a4d8205498c38e53fb85e25cf6428ead1e93e1 -> f16d28cc4cbab3b1c8d05d9e327676afafefdbdf ([pr/makewtx.11](https://github.com/ryanofsky/bitcoin/commits/pr/makewtx.11) -> [pr/makewtx.12](https://github.com/ryanofsky/bitcoin/commits/pr/makewtx.12))